src/sdk/pynni/nni/metis_tuner/metis_tuner.py

```diff
@@ -65,7 +65,7 @@ class MetisTuner(Tuner):
     """
 
     def __init__(self, optimize_mode="maximize", no_resampling=True, no_candidates=True,
-                 selection_num_starting_points=600, cold_start_num=10, exploration_probability=0.1):
+                 selection_num_starting_points=600, cold_start_num=10, exploration_probability=0.9):
```

Comment:
why changing from 0.1 to 0.9, seems like a big change.

Reply:
I talked with @xuehui1991 about it before. Metis Tuner will do more exploration with next_config.

Reply:
is it better to expose this configuration to users? or you think 0.9 is a good choice for different applications.

Reply:
Sure, I do think 0.9 is just a good choice.